config/prod.secret.exs

```diff
@@ -1,6 +1,6 @@
 use Mix.Config
 
-config :bors, BorsNG.Database.Repo,
+config :bors, BorsNG.Database.RepoPostgres,
```

Comment:
Is this project also released with MIX_ENV=prod? If yes, we should probably add a config for `BorsNG.Database.RepoMysql` here as well.

Reply:
Yes, bors does get released with MIX_ENV=prod. You can see that in the docker file

https://github.com/bors-ng/bors-ng/blob/master/Dockerfile#L25